core/src/avm2/globals/flash/events/MouseEvent.as

```diff
@@ -75,6 +75,10 @@ package flash.events
             return this.formatToString("MouseEvent","type","bubbles","cancelable","eventPhase","localX","localY","stageX","stageY","relatedObject","ctrlKey","altKey","shiftKey","buttonDown","delta");
         }
 
+        public function updateAfterEvent():void {
+            // TODO - determine when we should actually force a frame to be rendered.
```

Comment:
Can you use `log_warn` (add `import __ruffle__.log_warn;`) to add a stub message to this function?